docs/setup/install.md

```diff
@@ -17,6 +17,15 @@ section_menu_id: setup
 ## Using YAML
```

Comment:
Heading can be `Using Script` as description says "Stash can be installed via installer script....."